cmd/bisync/log.go

```diff
@@ -29,14 +29,23 @@ func (b *bisyncRun) indent(tag, file, msg string) {
 		logf = fs.Logf
 	}
 
+	if b.opt.DryRun {
+		logf = fs.Logf
+	}
+
 	if tag == "Path1" {
 		tag = Color(terminal.CyanFg, "Path1")
+		msg = strings.Replace(msg, "Queue copy to", Color(terminal.GreenFg, "Queue copy to Path2"), -1)
```

Comment:
What this would do now is:

```
NOTICE: - Path1    Queue copy to Path2 Path2 - file.txt
```

So the destination path is getting duplicated.

I'm not sure that any change is necessary here, since the info is already in the `msg`.